Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -80,6 +80,13 @@ class TabDisplayManager: NSObject {
         tabObservers = nil
     }
 
+    private func recordEventAndBreadcrumb(object: UnifiedTelemetry.EventObject, method: UnifiedTelemetry.EventMethod) {
+        let isTabTray = tabDisplayer as? TabTrayController != nil
+        let eventValue = isTabTray ? UnifiedTelemetry.EventValue.tabTray : UnifiedTelemetry.EventValue.topTabs
+        UnifiedTelemetry.recordEvent(category: .action, method: method, object: object, value: eventValue)
+        Sentry.shared.breadcrumb(category: "Tab Action", message: "object: \(object), action: \(method.rawValue), \(eventValue.rawValue), tab count: \(tabStore.count) ")
```

Comment:
I acknowledge this should be parameterized. I am doing this quickly for this specific case for now, and because I have no idea if this even works yet ;).